components/modal.js

```diff
@@ -28,8 +32,24 @@ export default function Modal({ userId, certificationNames }) {
       method: 'POST',
       body: JSON.stringify(formData)
     });
-    router.reload();
-    alert('Successfully Created Class');
+
+  
+    //router.reload();
+    //router.wait(10000);
+    setTimeout(function(){
+      window.location.reload();
+   }, 5000);
+    //alert('Successfully Created Class');
+
+    toast.success('Successfully Created Class!', {
```

Comment:
Looks like a lot of these arguments in the function are being repeated by our `<ToastContainer>`. See my comment on the toastcontainer in the page.